### PR TITLE
sapi: new verb

### PR DESCRIPTION
**Pull request by gverm**
This takes care of the last request in #1517.
It should also help to run apps that need SAPI in Win 7 mode.

### CODE REVIEW

**Comment by gverm** [comment]
I've converted this to a draft for now as it seems wine is unable to load the native dll, but I didn't see it before because I used native,builtin instead of native only.

**Comment by gverm** [comment]
This should be good to go now. Apparently this version of native SAPI needs to be in system32/syswow64. I changed the verb to remove the directory where builtin SAPI is located. I also put the native dll into system32/syswow64 now and changed the override to native only.

File: src/winetricks [hunk 1]
```diff
@@ -11919,6 +11919,37 @@ load_richtx32()
...
+    publisher="Microsoft" \
+    year="2011" \
+    media="download" \
+    conflicts="speechsdk" \
+    file1="../win7sp1/windows6.1-KB976932-X86.exe" \
+    installed_file1="${W_SYSTEM32_DLLS_WIN}/sapi.dll"
+
+load_sapi()
+{
+    for stub in "${W_SYSTEM32_DLLS}/Speech" "${W_SYSTEM64_DLLS}/Speech"; do
```
Comment (austin987) [inline src/winetricks:11933]: Please add a comment here about why this is needed.

Comment (gverm) [reply src/winetricks:11957]: @austin987 Done.

**Pull request merged**